decide_panel/src/components/Bar.js

```diff
@@ -0,0 +1,55 @@
+import Api from "../services/backend";
+import React from "react";
+import { Bar } from "react-chartjs-2";
+import Chart from 'chart.js/auto';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Full-Tortuga/decide-full-tortuga-almacenamiento/pullRequest?prid=8645779)
```suggestion
import Chart from "chart.js/auto";
```